packages/manager/modules/pci/src/projects/project/serving/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_serving_title": "Serving Engine",
+  "pci_projects_project_serving_description_label": "Description",
+  "pci_projects_project_serving_region_label": "Région",
+  "pci_projects_project_serving_add_label": "Créer un namespace",
+  "pci_projects_project_serving_view_label": "Détails",
+  "pci_projects_project_serving_delete_label": "Suppression du namespace"
```

Comment:
In PCI we usually write `Supprimer` in the datagrid actions instead of `Suppression du namespace`

Reply:
every comments solved: d84f5d0595339d4611672659f815f81a108948ce